src/components/timeline/Markers.js

```diff
@@ -229,9 +395,15 @@ class TimelineMarkersImplementation extends React.PureComponent<Props, State> {
           isSelected ? 'selected' : null
         )}
       >
-        <canvas
-          className="timelineMarkersCanvas"
-          ref={this._takeCanvasRef}
+        <MarkersCanvas
```

Comment:
This should probably be called `TimelineMarkerCanvas` to make it a bit clearer, and follow the practices in other panels.